app/javascript/components/batch_update/helpers/elasticsearch.js

```diff
@@ -78,6 +83,10 @@ export function buildElasticsearchQuery(args) {
     from: from || 0,
     size: size || defaultPageSize,
   }
+
+  window.$last_query = query
```

Comment:
🤔 ?

I guess the debug stuff from #193 would be handy here

Reply:
oh, yeah, that's my own mechanism for debugging similar to what you were doing. I attach things to `window` so I can grab them at will.